content/en/blog/2025/ottl-contexts-just-got-easier.md

```diff
@@ -0,0 +1,153 @@
+---
+title: OTTL contexts just got easier with context inference
+linkTitle: OTTL contexts just got easier
+date: 2025-02-17
+author: '[Edmo Vamerlatti Costa](https://github.com/edmocosta) (Elastic)'
+draft: true # TODO: remove this line once your post is ready to be published
+issue: 6289
+sig: Collector SIG
+cSpell:ignore: OTTL Vamerlatti
+---
+
+Selecting the right OTTL context for running statements can be challenging, even
+for experienced users. Choosing the correct option impacts both accuracy and
+efficiency, as using higher-level OTTL contexts can avoid unnecessary iterations
+through nested lower-level contexts.
+
+To simplify this process, the OpenTelemetry community is excited to announce
+OTTL
+[context inference](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md#context-inference)
+support for the
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor).
+This feature removes the need to manually specify OTTL contexts, improving
+statement processing efficiency by automatically selecting the most appropriate
+one. This optimization ensures that data transformations are both accurate and
+performant, allowing users to focus solely on their data without needing to
+understand the underlying concept of OTTL contexts.
```

Comment:
I want to highlight we're doing this to make it easier to use OTTL

```suggestion
This feature removes the need to for users to understand the underlying context concept of OTTL, instead allowing them to focus solely on their data. It also improves statement processing efficiency by automatically selecting the most appropriate conetxt. This optimization ensures that data transformations are both accurate and performant.
```

Reply:
Done, thanks!